Cue/app/common/CueHeader.ios.js

```diff
@@ -49,6 +49,7 @@ class CueHeader extends React.Component {
             numberOfLines={1}>
             {title}
           </Text>
+          {this.props.customTitleComponent}
```

Comment:
Note that due to the way the iOS header is laid out, the middle element only takes up about 50% of the width of the header container.